javascript/builtins/TypedArray.json

```diff
@@ -1213,19 +1213,23 @@
             "spec_url": "https://tc39.es/ecma262/multipage/indexed-collections.html#sec-properties-of-the-typedarray-constructors",
```

Comment:
This page (`TypedArray.name`) has been deleted. we should delete this entry as well. (It's a static property that's exactly equivalent to function names.)